package.json

```
@@ -1,8 +1,9 @@
 {
   "name": "voidbots",
-  "version": "2.1.0",
+  "version": "3.0.0",
```

```suggestion
  "version": "2.3.0",
```
Using semantic versioning (Semver) would be a better choice here since this branch does not include changes big enough to jump semver consideration.

---

Yeah, I suggested a dramatic bump because the class was being exported as a property, not the actual module.